terraform/bastion/Jenkinsfile

```diff
@@ -59,7 +82,7 @@ pipeline {
             script {
               dir("terraform/bastion") {
                 echo 'Initializing Terraform...'
-                sh "git clone https://github.com/nationalarchives/tdr-terraform-modules.git"
+                sh "git clone --single-branch --branch add-bastion-user-lambda https://github.com/nationalarchives/tdr-terraform-modules.git"
```

Comment:
I'm guessing this is temporary and should be removed before merging?

Reply:
Yep, good spot, cheers. I've taken this out.